package it.csi.sigit.sigitbatchn.business.dao.sigitbatchn.exceptions;

/**
 * Le eccezioni rilanciate dai metodi del DAO sigitTConsumo sono di questo tipo.
 * @generated
 */
public class SigitTConsumoDaoException extends DaoException {
	/** 
	 * @param message
	 * @generated
	 */
	public SigitTConsumoDaoException(String message) {
		super(message);
	}

	/**
	 * @param message
	 * @param cause
	 * @generated
	 */
	public SigitTConsumoDaoException(String message, Throwable cause) {
		super(message, cause);
	}

}
